docs/environment_setup.md

```diff
@@ -68,6 +68,10 @@ Use `flytectl` to start a demo Flyte cluster:
 flytectl demo start
 ```
 
+After this completes, be sure to export the Flyte config as it will be essential later. Run the command in the output that looks like this: 
+```export FLYTECTL_CONFIG=/Users/guestaccount/.flyte/config-sandbox.yaml
+```
```

Comment:
```suggestion
```{prompt} bash
export FLYTECTL_CONFIG=/Users/guestaccount/.flyte/config-sandbox.yaml
```
```

Reply:
Also, I thought that we automatically use the sandbox config at least for flytekit remote stuff. Is this not actually true?

Reply:
I'm not sure on that part. But this was the fix for the issues I was running into :)